kitsune/upload/views.py

```diff
@@ -75,6 +75,11 @@ def del_image_async(request, image_id):
         message = _("You do not have permission to do that.")
         return HttpResponseForbidden(json.dumps({"status": "error", "message": message}))
 
+    try:
```

Comment:
Maybe you could check against the `content_type` you are interested in and call this method instead of checking for an exception. It would be clearer on which type you are operating on

Reply:
I quite like the duck typing here, but could add a comment explaining where this is currently used